src/Console_TEST.cc

```diff
@@ -25,7 +25,8 @@
 
 #include "test/util.hh"
 
-#ifndef _WIN32
+namespace igncmn = ignition::common;
```

Comment:
These abbreviations can make code harder to find and search-and-replace later on. 

How about `using namespace ignition::common;`?

Reply:
Yeah, good idea. I've just found `igncmn` in more places, so I thought it is kind of standard.